test.cfg

```diff
@@ -52,10 +52,17 @@ HUEY_SCHEDULE = {
     "check_latest_es_backup": CRON_NEVER,
     "prune_es_backups": CRON_NEVER,
     "public_data_dump" : CRON_NEVER,
-    "harvest": {"month": "*", "day": "*", "day_of_week": "*", "hour": "5", "minute": "30"}
+    "harvest": {"month": "*", "day": "*", "day_of_week": "*", "hour": "5", "minute": "30"},
+    "anon_export": CRON_NEVER,
 }
 
 # =======================
 # Account settings
 PUBLIC_REGISTER = True
 LOGIN_VIA_ACCOUNT_ID = True
+
+#########################################################
+# Background tasks --- anon export
+TASKS_ANON_EXPORT_CLEAN = False
+TASKS_ANON_EXPORT_LIMIT = None
+TASKS_ANON_EXPORT_BATCH_SIZE = 100000
```

Comment:
Since these properties don't change the defaults in `settings.py` I think they can be omitted